qpp-data-restoration/pom.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>gov.cms.qpp.conversion</groupId>
```

Comment:
Do we want to specify the parent pom?  Not entirely sure.

Reply:
Agreed, unless we're using something from the parent POM...  I see this as a one-off module that encapsulates a conversion app.

Reply:
Let me do a push and see where the code is. I think I forgot to do one more push of tweaks before submitting the PR.

Reply:
Yeah... the parent pom wasn't in my version lol. That's my fault.